### PR TITLE
LUI-153: Footer to show optional extra data from runtime properties.

### PR DESCRIPTION
**Pull request by HerbertYiga**
https://issues.openmrs.org/browse/LUI-153

Footer shows an optional extra data from run time properties.
Also added a screen short on the ticket ID

### CODE REVIEW

File: omod/src/main/webapp/template/footerFull.jsp [hunk 1]
```diff
@@ -45,6 +45,8 @@
 			
 			<span id="codeVersion"><openmrs:message code="footer.version"/>: ${openmrsVersion}</span>
 			
+			<span id="dockerImage"><openmrs:message code="legacyui.footer.extradata"/>: ${openmrsVersion}</span>
```
Comment (mks-d) [inline omod/src/main/webapp/template/footerFull.jsp:48]: This `code` isn't necessary.

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -6,6 +6,7 @@ legacyui.manageuser.providerAccount=Provider Account
 legacyui.manageuser.createProviderAccount=Create a Provider account for this user
 legacyui.manageuser.providerIdentfier=Provider Identifier(s):
 legacyui.manageuser.noProviderIdentifier=No Identifier Specified
+legacyui.footer.extradata=Docker Image
```
Comment (mks-d) [inline api/src/main/resources/messages.properties:9]: This message should go away.

File: omod/src/main/webapp/template/footerFull.jsp (hunk 1 above)
Comment (mks-d) [inline omod/src/main/webapp/template/footerFull.jsp:48]: In the end you'll have something like this:
```html
<span id="extraData">${footerExtraData}</span>
```
However this span should be remain hidden if `${footerExtraData}` is blank or empty.

**Comment by HerbertYiga** [comment]
@mks-d  okie,i have got the concept and let me make the changes

**Pull request closed without merging**